R/html_dependency.R

```diff
@@ -231,32 +231,37 @@ copyDependencyToDir <- function(dependency, outputDir, mustWork = TRUE) {
     }
   }
 
-  if (!file.exists(outputDir))
+  if (!dir_exists(outputDir))
     dir.create(outputDir)
```

Comment:
A simpler alternative is to remove the existence check and do `dir.create(outputDir, showWarnings = FALSE)`